easybuild/easyblocks/a/amber.py

```diff
@@ -125,11 +126,11 @@ def install_step(self):
         if mklroot:
             env.setvar('MKL_HOME', mklroot)
         elif openblasroot:
-	    lapack = os.getenv('LIBLAPACK')
-	    if lapack is None:
-		raise EasyBuildError("LIBLAPACK (from OpenBLAS) not found in environement.")
-	    else:
-		env.setvar('GOTO', lapack)
+            lapack = os.getenv('LIBLAPACK')
+            if lapack is None:
+                raise EasyBuildError("LIBLAPACK (from OpenBLAS) not found in environement.")
```

Comment:
Since you are at it, while not fix typos like `environement`?

Reply:
as you wish :-)